test/utils.ts

```diff
@@ -66,3 +68,11 @@ export function runInstanceTestCases<InstanceType extends {[key: string]: any}>(
     }
   }
 }
+
+export function maybeWaitForRuntime(callback: () => void): void {
```

Comment:
To run some steps before running any test code mocha has some strategies, can you check if these ones play nice with web? Would definitely make the setup simpler.
- https://mochajs.org/#root-hook-plugins
- https://mochajs.org/#global-setup-fixtures

Reply:
Thanks for pointing this out, I was unaware!